nixos/modules/services/databases/lldap.nix

```diff
@@ -224,6 +548,28 @@ in
         + ''
           ${lib.getExe cfg.package} run --config-file ${format.generate "lldap_config.toml" cfg.settings}
         '';
+      postStart = ''
+        export LLDAP_URL=http://127.0.0.1:${toString cfg.settings.http_port}
+        export LLDAP_ADMIN_USERNAME=${cfg.ensureAdminUsername}
+        export LLDAP_ADMIN_PASSWORD=${
+          if cfg.ensureAdminPassword != null then cfg.ensureAdminPassword else ""
+        }
```

Comment:
Setting this will mean, that an admin password is written to `/nix/store`. In this case it will even be linked to `/etc/systemd`. I'd not even treat this as a warning, but not support it outright.

Reply:
I agree, this we shouldn't support this.